zerver/actions/realm_settings.py

```diff
@@ -267,6 +267,7 @@ def do_deactivate_realm(realm: Realm, *, acting_user: Optional[UserProfile]) ->
     if realm.deactivated:
         return
 
+    do_send_realm_deactivation_email(realm, acting_user)
```

Comment:
Should we call this function after the database operations for deactivation are completed?

Reply:
Agreed, Revert that line after db operations.